subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/EclipsePlugin.java

```diff
@@ -300,6 +300,7 @@ public void execute(ScalaBasePlugin scalaBasePlugin) {
 
                 // exclude the dependencies already provided by SCALA_CONTAINER; prevents problems with Eclipse Scala plugin
                 project.getGradle().projectsEvaluated(gradle -> {
+                    // TODO: support Scala3
```

Comment:
Eclipse will not currently work anyway :/ so no need to update this.

Reply:
:)